### PR TITLE
Add low income credit

### LINKED ISSUE #10: Add Italy personal income tax low-income exemptions

### PR DESCRIPTION
**Pull request by anth-volk**
Fixes #10 
Adds low-income credit

### CODE REVIEW

File: policyengine_it/parameters/gov/agenzia_delle_entrate/credits/low_income/eligibility.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: Italy limits low income tax credit eligibility to individuals making less than or equal to this threshold per year, depending on employment status.
+metadata:
+  unit: currency-EUR
```
Comment (PavelMakarchuk) [inline policyengine_it/parameters/gov/agenzia_delle_entrate/credits/low_income/eligibility.yaml:3]: we need to a breakdown metadata piece:
 e.g.:
   breakdown:
    - [variable_class_name]

File: policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        print(is_eligible)
+        exemption_rate = parameters(
+            period
+        ).gov.agenzia_delle_entrate.credits.low_income.exemption_rate
+        print(exemption_rate)
+        income = person("total_individual_pre_tax_income", period)
+        print(income)
+        exempted_income = exemption_rate * income
+        print(exempted_income)
+        print(is_eligible * exempted_income)
```
Comment (PavelMakarchuk) [inline policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py:25]: delete print

File: policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+from policyengine_it.model_api import *
+
+from policyengine_it.variables.household.person.employment_category import (
```
Comment (PavelMakarchuk) [inline policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py:3]: dont need to import here as we will use it in the var

File: policyengine_it/variables/gov/agenzia_delle_entrate/tax/income/income_tax_before_refundable_credits.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class income_tax_before_refundable_credits(Variable):
+    value_type = float
+    entity = Person
+    definition_period = YEAR
+    unit = EUR
+    label = "Italian national income tax before refundable credits"
+    documentation = "Income tax liability (including other taxes) after non-refundable credits are used, but before refundable credits are applied"
+    adds = [
```
Comment (PavelMakarchuk) [inline policyengine_it/variables/gov/agenzia_delle_entrate/tax/income/income_tax_before_refundable_credits.py:11]: this should be capped at 0 if non refundable

Comment (PavelMakarchuk) [reply policyengine_it/variables/gov/agenzia_delle_entrate/tax/income/income_tax_before_refundable_credits.py:11]: formula:
    amount = household(....
    non_ref_c = household(...
return max_(amount - non_ref_c, 0)

File: policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py (hunk 2 above)
Comment (anth-volk) [reply policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py:25]: Fixed in a previous PR

File: policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py (hunk 3 above)
Comment (anth-volk) [reply policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py:3]: Fixed in a previous PR

**Pull request merged**